src/network/networkd-network-gperf.gperf

```diff
@@ -110,6 +110,9 @@ Bridge.AllowPortToBeRoot,               config_parse_bool,
 Bridge.UnicastFlood,                    config_parse_bool,                              0,                             offsetof(Network, unicast_flood)
 BridgeFDB.MACAddress,                   config_parse_fdb_hwaddr,                        0,                             0
 BridgeFDB.VLANId,                       config_parse_fdb_vlan_id,                       0,                             0
+BridgeVLAN.PVID,                        config_parse_uint32,                            0,                             offsetof(Network, pvid)
```

Comment:
hmm, what's the size of a VLAN ID? you use config_parse_uint32 here, which implies it's a 32bit nr?

The variable is defined as "unsigned" however. Which of course effectively the same as uint32_t on all archs we care about, but I think we really should stick to one type.

After checking the ethernet specs it appears the actual range for vids is 1..4094, i.e. 12 bits, but excluding the special values 0 and 0xFFF.  Which makes me think that we should really always us "uint16_t" as a type here.